jdaviz/core/template_mixin.py

```diff
@@ -1294,17 +1294,20 @@ def _on_layers_changed(self, msg=None):
                 layer for viewer in viewers
                 for layer in getattr(viewer, 'layers', [])
                 # don't include WCS-only layers unless asked:
-                if (
-                    not hasattr(layer.layer, 'meta') or
-                    (not layer.layer.meta.get('_WCS_ONLY', False))
+                if not hasattr(layer.layer, 'meta') or (
+                    hasattr(layer.layer, 'meta') and
```

Comment:
I think we need that line because we need to check if the layer is a WCS-only layer, and the flag is in the metadata, which we can only check if the metadata are present.

Reply:
But if it ever get past the first `or`, it is guaranteed to have `meta` already.

Reply:
True!